src/classes/RecipeRepository.js

```diff
@@ -1,7 +1,18 @@
 class RecipeRepository {
-  constructor() {
-
+  constructor(recipeData) {
+    this.recipeObjects = recipeData;
     // One class to get you started!
```

Comment:
Check for the following: 
- [ ] Is the code following the SRP and is it dry? 
- [ ] Are there any console Logs? 
- [ ] Is there any commented out code or pseudocode?